index.html

```diff
@@ -6585,7 +6585,8 @@ <h5>Presentational Roles Conflict Resolution</h5>
 			<div class="role-description">
 				<p>A perceivable <rref>section</rref> containing content that is relevant to a specific, author-specified  purpose and sufficiently important that users will likely want to be able to navigate to the section easily and to have it listed in a summary of the page. Such a page summary could be generated dynamically by a user agent or assistive technology.</p>
 				<p>Authors SHOULD limit use of the region role to sections containing content with a purpose that is not accurately described by one of the other <rref>landmark</rref> roles, such as <rref>main</rref>, <rref>complementary</rref>, or <rref>navigation</rref>.</p>
-				<p>Authors MUST give each element with role region a brief label that describes the purpose of the content in the region. Authors SHOULD reference a visible label with <pref>aria-labelledby</pref> if a visible label is present. Authors SHOULD include the label inside of a heading whenever possible. The heading MAY be an instance of the standard host language heading element or an instance of an element with role <rref>heading</rref>.</p>
+				<p>Authors MUST give each element with role region a brief label that describes the purpose of the content in the region. Authors SHOULD use a visible heading to provide a label whenever possible.  The heading MAY be an instance of the standard host language heading element or an instance of an element with role <rref>heading</rref>.  For host languages that support elments with the default role of region (e.g. HTML <code>section</code>), an explict role=<q>region</q> attribute must be defined to enable the heading labeling technique.</p>
```

Comment:
Regarding the "For host languages...." bit, might this be reworded as:
>An explicit `role="region"` attribute must be defined to enable the heading labeling technique.

Since `section` does *not default* to `role=region`, unless given an `aria-label` or `aria-labelledby`, which would then take precedent over this technique.